app/models/service_ansible_playbook.rb

```diff
@@ -45,6 +45,12 @@ def postprocess(action)
     delete_inventory(action) unless use_default_inventory?(hosts)
   end
 
+  def orchestration_stacks
```

Comment:
A second thought - the method should not return a hash. Just like `vms`, an array is better. And this method can be moved to `Service` thus good for Orchestration and AnsibleTower services too.